src/Microsoft.AspNetCore.Razor.LanguageServer/Microsoft.AspNetCore.Razor.LanguageServer.csproj

```diff
@@ -19,17 +19,14 @@
     <PackageReference Include="Microsoft.AspNetCore.Blazor.Razor.Extensions" Version="$(MicrosoftAspNetCoreBlazorExtensionsPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!-- We need this to compile but don't want it referenced by the project. -->
-    <ProjectReference Include="..\Microsoft.AspNetCore.Razor.OmniSharpPlugin\Microsoft.AspNetCore.Razor.OmniSharpPlugin.csproj">
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <Targets>Publish</Targets>
-    </ProjectReference>
-  </ItemGroup>
-
   <Target Name="PublishLanguageServerNativeExecutables" AfterTargets="Pack" DependsOnTargets="PublishAllRids" />
 
   <Target Name="IncludeOmniSharpPlugin" AfterTargets="Publish" Condition="Exists('$(PublishDir)')">
```

Comment:
Does this really only need to happen on `Publish`? Isn't it also needed during development? Or are you assuming that developers always start by running `build.cmd` which implictly does a publish (not just a build)?